client/src/containers/SignIn/SignInForm/SignInForm.tsx

```diff
@@ -0,0 +1,62 @@
+import React from "react";
+import FormHelperText from "@material-ui/core/FormHelperText";
+import styled from "styled-components";
+import { Formik, Form, Field } from "formik";
+import FormTextField from "../../../components/FormTextField/FormTextField";
+import * as Yup from 'yup';
+import { ButtonFormPrimaryMUI } from "../../../components/Button/Button";
+import FormCheckboxField from "../../../components/FormCheckboxField/FormCheckboxField";
+
+const StyledFormikForm = styled(Form)`
+    display: grid;
+    grid-gap: 15px;
+    width: 100%;
+    max-width: 400px;
+    padding: 0 20px;
+`
+
+const SigninSchema = Yup.object().shape({
+  email: Yup.string()
+    .email('Invalid email')
+    .required('Email is required'),
+    password: Yup.string()
+    .min(6, 'Password length must be at least 6 characters')
+    .required('Password is required'),
+});
+
+interface Values {
+  email: string;
+  password: string;
+  remember: boolean;
+}
+
+interface Props {
+  onSubmit: (values: Values) => void;
+}
+
+const SignIn: React.FC<Props> = ({onSubmit}) => {
+  return (
+ 
+      <Formik
```

Comment:
how this is working with server side errors ?

Reply:
it hasn't build in feature for handle outside errors, but there are some workarounds which i'll try use